cmd/ocm/cluster/create/create.go

```diff
@@ -67,11 +72,28 @@ func init() {
 		"osd-4",
 		"The OCM flavour to create the cluster with",
 	)
+	fs.StringVar(
+		&args.expirationTime,
+		"expiration-time",
+		args.expirationTime,
+		"Specified time when cluster should expire (RFC3339). Only one of expiration-time / expiration may be used.",
+	)
+	fs.DurationVar(
+		&args.expirationSeconds,
+		"expiration",
+		args.expirationSeconds,
+		"Expire cluster after a relative duration like 2h, 8h, 3d. Only one of expiration-time / expiration may be used.",
+	)
```

Comment:
Is there any specific use-case for supporting both flags?

I'm wondering what people would use the most and settling on that - you can always do, for example
```
ocm cluster create --expiration-timestamp=$(date -Iseconds -d+7days)
```
If the goal of supporting both formats is for scripting purposes, then it can be done like that example. Otherwise we can just document one of them that we're more likely to use. I would personally use this a format lot more often:
```
ocm cluster create --expiration=3d
```
but the timestamp approach might be a lot more scriptable, and so might make more sense to support that one.

Reply:
I see now, makes sense. Thanks for the explanation :+1: